.github/workflows/node.js.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: Node.js CI
+name: Node.js Test CI
```

Comment:
Well, it's not only testing but also releasing.
So I would just have kept "Node CI"